bin/travis.sh

```diff
@@ -4,6 +4,11 @@
 if [ "$1" == 'before' ]; then
 	cd "$WP_CORE_DIR/wp-content/plugins/woocommerce-admin/"
 	npm run build:feature-config
+<<<<<<< HEAD
```

Comment:
Conflict here.

Reply:
😱

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/5558/commits/27d3fcd04c08cafe4237ef65f9143c3534d08861 I also added some new context to that comment for the next person to know when they can remove that code.